### PR TITLE
Upgrade to latest Yaru default

### PR DESCRIPTION
**Pull request by clobrano**
This change supports the new Yaru folder structure.
Currently it installs only the default "ambiance" variant.

### CODE REVIEW

File: org.gtk.Gtk3theme.Yaru.json [hunk 1]
```diff
@@ -70,10 +70,13 @@
...
-              "sed -i -e \"s/gtk-3.20/gtk-3.0/\" gtk/src/light/gtk-3.20/meson.build",
-              "rm -r gtk/src/light/gtk-3.0",
-              "mv gtk/src/light/gtk-3.20 gtk/src/light/gtk-3.0"
+              "sed -i -e \"s/subdir('dark')//\" gtk/src/meson.build",
+              "sed -i -e \"s/subdir('light')//\" gtk/src/meson.build",
+              "rm -r gtk/src/default/gtk-3.0",
+              "mv gtk/src/default/gtk-3.20 gtk/src/default/gtk-3.0",
+              "sed -i -e \"s/subdir('gtk-3.0')//\" gtk/src/default/meson.build",
+              "sed -i -e \"s/subdir('gtk-3.20')/subdir('gtk-3.0')/\" gtk/src/default/meson.build",
+              "sed -i -e \"s/gtk-3.20/gtk-3.0/\" gtk/src/default/gtk-3.0/meson.build"
```
Comment (TingPing) [inline org.gtk.Gtk3theme.Yaru.json]: @clobrano Since you contribute upstream why are you not just fixing these instead of adding more hacks?

Comment (clobrano) [reply org.gtk.Gtk3theme.Yaru.json]: Fixing what?

Comment (TingPing) [reply org.gtk.Gtk3theme.Yaru.json]: Building this package should require zero uses of `sed`.

Comment (clobrano) [reply org.gtk.Gtk3theme.Yaru.json]: Indeed, it should be possible

**Pull request merged**